src/app/builder/builder.component.html

```diff
@@ -50,37 +50,46 @@
 
     <form #builderForm="ngForm">
       <section class="builder-step" id="step-3" *ngIf="props">
-        <h4>Design Your Player</h4>
+        <h4>Customize Your Player</h4>
+        <p>These fields are for overrides only. They should be set
+          if you want to override the episode's information in your RSS feed
+          for the player.
+        </p>
 
         <fieldset>
           <label for="title">Title</label>
+          <p>If you leave this blank, the player will default to  <em>{{this.defaults.title}}</em>.</p>
```

Comment:
@cavis good points all. So, do you mean an actual placeholder in the input box? (Was avoiding that because of this in the original ticket: `Don't fill in the form fields by default`)